src/Orleans/Runtime/Constants.cs

```diff
@@ -36,6 +36,7 @@ internal class Constants
         public static readonly GrainId TypeManagerId = GrainId.GetSystemTargetGrainId(17);
         public static readonly GrainId ProviderManagerSystemTargetId = GrainId.GetSystemTargetGrainId(19);
         public static readonly GrainId DeploymentLoadPublisherSystemTargetId = GrainId.GetSystemTargetGrainId(22);
+        public static readonly GrainId ClusterDirectoryServiceId = GrainId.GetSystemTargetGrainId(24);
        
```

Comment:
Yes, this should be part of the branch geo_globalsingleinstance. I will move it.